### DISCUSSION TITLE
0.14 behavior of "Show Reviewed" toggle. Remember last value or default to On

### DISCUSSION THREAD

**Discussion by robwaldhauser** [blakeblackshear/frigate#11670] [Ideas]
The new layout is absolutely gorgeous, but one thing that has been frustrating is going to the Review tab and not seeing everything that's happening on my cameras. To remember to and have to click the "Show Reviewed" toggle every time the page loads (even if just closing a preview I was watching) is very tedious and not very user friendly. I believe the default option would be best to be turned on, and if not, at least the ability to change that default behavior or remember the last value would be appreciated.

![image](https://github.com/blakeblackshear/frigate/assets/83437058/4c718c5f-9904-44e3-92a1-4e95d90384c6)

**Comment by NickM-27** [comment]
Just curious, is there a reason why you want to see things that you have already reviewed? If you already looked at it and decided it did not require action, what is the point of seeing it again?

In the old version of frigate one very common feature request was that people didn't want to keep seeing events that they already reviewed. It makes it easy to see only things that might matter still because previously reviewed things were determined to not matter. I would suggest that it is user friendly, maybe your use case is just different so it would be good to understand your use case better.

**Reply by robwaldhauser** [reply]
There are a lot of instances where I click the event in the live view and as new events happen I move to the review tab to get a visual timeline of the activity. My definition of "Review" might be a bit different than what the intent was originally for that tab. If that's the case, I think then maybe if I toggle "Show Reviewed" it should atleast stay toggled on while I'm clicking through previews on the page instead of hiding everything than forcing me to retoggle it and find my place scrolling through everything again.

Like I said, I might be using it a little different then intended, but I have been enjoying visually seeing a timeline of the day through the previews of the day. We just recently had a bunch of trespassing from nearby asphalt layers at a neighbors house, so I was tracking their enters and exits throughout the day they were cutting through our property in in very inappropriate ways.

**Reply by NickM-27** [reply]
> I think then maybe if I toggle "Show Reviewed" it should atleast stay toggled on while I'm clicking through previews on the page instead of hiding everything than forcing me to retoggle it and find my place scrolling through everything again.

sounds like you might be using an old version or maybe navigating in a weird way. On the latest version if you have any filters (including the reviewed toggle) set and you click on a review item then navigate back it will retain your filters and it will scroll the page down to where you were before

**Reply by robwaldhauser** [reply]
Ah, I really feel bamboozled right now. The last few days I really feel like the behavior was what I had stated but I'm doing it again now and it's behaving like you said.... I'm not that old yet lol.

So back to what you are mentioning, in my use, just because I view something doesn't mean it doesn't require action or that I don't want to see it again. Is there a way for me to not have it automatically mark viewed items as "Reviewed"?

**Reply by NickM-27** [reply]
No, there's no way for that right now.

**Reply by robwaldhauser** [reply]
After I posted I found a better way of describing my use case, while I use Frigate for notifications, I don't always review all notifications (partially because I still haven't worked out all the false positives) so my use of Frigate is as a way to quickly get a visual log of activity on the property. That might better describe how I use it and how it would be useful for me.

**Reply by robwaldhauser** [reply]
> No, there's no way for that right now.

Ah i think that is my suggestions/idea then :) Thank you!

**Reply by NickM-27** [reply]
You can leave a 👍 on https://github.com/blakeblackshear/frigate/issues/11621

**Reply by robwaldhauser** [reply]
> You can leave a 👍 on #11621

Oh I tried searching for this but never saw it, that's perfect. Thanks so much, and awesome work to the team!
